test/integration/amazon.yml

```diff
@@ -1,6 +1,7 @@
 - hosts: amazon
   gather_facts: true
   roles:
+    - { role: test_s3_bucket, tags: test_s3_bucket }
```

Comment:
Now that we're using `ansible-test` to run AWS integration tests, we should consider removing this file instead of updating it. Is there a reason we should keep it around? If so, we need to update the change classification in `ansible-test` so that changes to this file don't trigger all integration tests.